addon/serializer.js

```diff
@@ -27,10 +27,34 @@ import Ember from 'ember';
 export default Ember.Object.extend({
 
   /**
-    The `store` property is the application's `store` that contains all records.
-    It's injected as a service.
-    It can be used to push records from a non flat data structure server
-    response.
+    The `store` property is the application's `store` that contains
```

Comment:
so it's a bit confusing here because you have the store passed into the function's parameter list, and also `this.store`, which gets set when the serializer is instantiated.

I think this documentation is for `this.store` because of `@property`, but isn't used in the example below.